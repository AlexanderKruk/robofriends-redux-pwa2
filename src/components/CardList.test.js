import { shallow } from "enzyme";
import CardList from "./CardList";

it("Testing CardList", () => {
  const mockRobots = [
    {
      id: 1,
      name: "John",
      surname: "Snow",
      email: "john.snow@gmail.com",
    },
  ];
  expect(<CardList robots={mockRobots} />).toMatchSnapshot();
});
