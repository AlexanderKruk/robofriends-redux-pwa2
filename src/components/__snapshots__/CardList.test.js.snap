// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing CardList 1`] = `
<CardList
  robots={
    Array [
      Object {
        "email": "john.snow@gmail.com",
        "id": 1,
        "name": "John",
        "surname": "Snow",
      },
    ]
  }
/>
`;
